mmv1/products/workstations/WorkstationCluster.yaml

```
@@ -60,6 +60,18 @@ examples:
     min_version: 'beta'
     vars:
       workstation_cluster_name: 'workstation-cluster-custom-domain'
+  - name: 'workstation_cluster_tags'
+    primary_resource_id: 'default'
+    min_version: 'beta'
+    vars:
+      workstation_cluster_name: 'workstation-cluster-tags'
+      key_short_name: 'keyname'
+      value_short_name: 'valuename'
+      org_id: '123456789'
+    test_vars_overrides:
+      'key_short_name': '"tf-test-key-" + acctest.RandString(t, 10)'
+      'value_short_name': '"tf-test-value-" + acctest.RandString(t, 10)'
+      'org_id': 'envvar.GetTestOrgFromEnv(t)'
```

See https://googlecloudplatform.github.io/magic-modules/reference/resource/#examples for more details.

```suggestion
    test_env_vars:
      org_id: "ORG_ID"
    test_vars_overrides:
      'key_short_name': '"tf-test-key-" + acctest.RandString(t, 10)'
      'value_short_name': '"tf-test-value-" + acctest.RandString(t, 10)'
```